ocaml/xapi/xapi_host.ml

```diff
@@ -2629,14 +2636,31 @@ let get_sched_gran ~__context ~self =
 let emergency_disable_tls_verification ~__context =
   (* NB: the tls-verification state on this host will no longer agree with state.db *)
   Stunnel_client.set_verify_by_default false ;
-  Unixext.unlink_safe Xapi_globs.verify_certificates_path
+  Unixext.unlink_safe Xapi_globs.verify_certificates_path ;
+  try
+    (* we update the database on a best-effort basis because we
+       might not have a connection *)
+    let self = Helpers.get_localhost ~__context in
+    Db.Host.set_tls_verification_enabled ~__context ~self ~value:false
+  with e ->
+    error "Failed to update database after TLS verication was disabled: %s"
```

Comment:
I think downgrading this to warning or even info is warranted since the exception is expected.